thunder/dynamo/compiler.py

```diff
@@ -1,9 +1,16 @@
+from typing import List, Dict
```

Comment:
I'm afraid in this repo I feel it not common to see `from typing import List, Dict, Optional, Tuple, Set`, could you audit the imports? guessing these could be auto-imported by some editor or completion plugins though